packages/web/src/js/__tests__/SegmentedControl.test.ts

```diff
@@ -0,0 +1,73 @@
+import { cssVariablePrefix } from '@lmc-eu/spirit-design-tokens';
+import { clearFixture, getFixture } from '../../../tests/helpers/fixture';
+import SegmentedControl from '../SegmentedControl';
+
+describe('SegmentedControl', () => {
+  let fixtureEl: HTMLElement;
+  let fixtureControl: HTMLElement;
+
+  beforeAll(() => {
+    fixtureEl = getFixture();
+  });
+
+  afterEach(() => {
+    clearFixture();
+  });
+
+  beforeEach(() => {
+    fixtureEl.innerHTML = `
```

Comment:
Cool! 👍🏻  Thank you for reducing the fixture to a minimum example, this is what I meant 🙂.